### ISSUE TITLE
Enable sharing of multiple notes at once

### ISSUE THREAD

**Issue by scaidermern** [osmandapp/Osmand#5558]
It is already possible to share multiple favorites and OSM notes/edits in "My Places". Tracks however can only be shared one by one. This is tedious, especially when using the track recording feature and having individual tracks for each day. Please add a share button for tracks similar to the share buttons for favorites and OSM notes/edits in "My Places".

**Comment by quazgar** [comment]
I would also very much appreciate this feature.

**Comment by vshcherb** [comment]
So far it's supported as Export package! We doubt what format should we package it cause there is no standard. BTW *.osf file is a simple zip which could be renamed and unpacked

**Comment by scaidermern** [comment]
True, however this will export _all_ tracks which might be a significant amount of data. It is already possible to share multiple, individual favorites at once. So why not add it for tracks, too?

Personally I need this very often for mapping. I record multiple tracks, then I want to export them to open them in JOSM. This just applies to a handful of tracks. I definitely don't want to export _all_ of them.

**Comment by pl71** [comment]
Nice to have faeture.

**Comment by gubrist** [comment]
My expectation is a device-independent *.GPX File, or some files compressed in a *. ZIP File.

**Comment by vshcherb** [comment]
We might want to have another format for *.zip to allow to export Media with tracks https://github.com/osmandapp/OsmAnd/issues/7125
